<ui version="4.0" >
 <class>folderviewPreviewConfig</class>
 <widget class="QWidget" name="folderviewPreviewConfig" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Check the file types you want icon previews for:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="listView" >
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
